_drafts/community.md

```diff
@@ -0,0 +1,73 @@
+---
+name: community
+layout: post_discourse
+title: Hey! You there! You are welcome here.
+authors:
+  - name: Shannon E. Ellis
+    url: https://shanellis.github.io/
+categories:
+  - blog
+topicid: ???
+tags:
+  - unconf
+  - community
+  - meetings
+---
```

Comment:
@ShanEllis Karthik says it's a great post and we'd love to make it live this Friday June 23! (might go up Thurs night). His comment below shows how to make the emoji pic show up.

Please tag me and @sckott when you've made that change and consider the one comment below. Scott will merge your pull request and make it live and I'll tweet about it Friday morning!
Thank you so much for taking the initiative on this and "making it real" by doing the extra work a post requires. Now you can get a little twitter recognition for your effort :-)